frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.ts

```diff
@@ -62,28 +74,35 @@ export const getDatasetKey = (
  * @param {DatasetColumn[]} columns - The columns of the raw dataset.
  * @param {RowValue[]} row - The raw row of values.
  * @param {number} cardId - The ID of the card.
- * @param {number} [breakoutIndex] - The breakout column index for charts with two dimension columns selected.
+ * @param {number} dimensionIndex - The dimension column index.
+ * @param {number} breakoutIndex - The breakout column index for charts with two dimension columns selected.
  */
-const aggregateMetricsForDatum = (
+const aggregateColumnValuesForDatum = (
```

Comment:
This is for tooltips where we currently show non-metric values from the dataset that are not a dimension or a breakout